templates/deployment_backend.yaml

```diff
@@ -67,17 +67,29 @@ spec:
           - name: LICENSE_KEY
             valueFrom:
               secretKeyRef:
+                {{- if .Values.config.licenseKeySecretName }}
+                name: {{ .Values.config.licenseKeySecretName }}
+                {{- else }}
                 name: {{ template "retool.fullname" . }}
+                {{- end }}
                 key: license-key
```

Comment:
should I change the keys to camelcase? It will be harder to change later since this is not currently exposed right now as is.

Reply:
Changed this to be configurable